CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 Framer Motion adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.1.0] 2020-07-16
+
+### Added
+
+-   Added `layoutX` and `layoutY` readonly MotionValue props for hooking other MotionValues into the layout calculations.
+
+### Fixed
+
+-   Fixed `originX` and `originY` being used incorrectly in bounding box measurements for layout children.
```

Comment:
I can't seem to see the code that effects this? 🤔

Reply:
https://github.com/framer/motion/pull/633/files#diff-1838108ae6ed35972d15efc143a8197bR102

Reply:
Ahh 👍